frontend/src/app/services/payment.service.ts

```diff
@@ -81,4 +81,7 @@ export class PaymentService {
     return this.http.get<Payment>(`${environment.baseUrl}/payments/${id}`);
   }
 
+  updateValidated(id: string): Observable<Payment> {
```

Comment:
Looking to this name anyone should see the implementation to understand what's the meaning of "updateValidated" ... please change the name to something like "validatePayment" or "setValidatedToTrue" (I prefer the first option).

Remember your code must be auto-documented

Reply:
Ok, i did